test/e2e/testkit/otlp/matchers/log_matchers.go

```diff
@@ -0,0 +1,178 @@
+package matchers
```

Comment:
Let's add some unittests for these matchers as well (same as for the existing matchers)

Reply:
unit test added